tools/hangar/__snapshots__/test_corpus/valid/bucket_events.w.ts.snap

```diff
@@ -1412,49 +1417,73 @@ class $App extends $AppBase {
     }
   }
 }
-
-new $App().synth();"
+new $App().synth();
+"
 `;
 
-exports[`wing compile -t tf-aws > proc.4c20a426ba35d2bc9530ee9673d6111365502af4b9943b5993189fef4c9cbb24/index.js 1`] = `
-"async handle(key) { const {  } = this; {
-  {console.log(\`updated \${key}\`)};
-} };"
+exports[`wing compile -t tf-aws > proc.0b6500d3229de22066b0d8ef0d85b80986dabd1f05d4adce56d1a2f16b46a58e/index.js 1`] = `
+"async handle(key) {
+  const { other } = this;
+  {
```

Comment:
@Chriscbr any idea why we are surrounding the body of handlers in an an inner block? I don't think that's providing any value.

Reply:
Agree, I think it's just an implementation bug / shortcoming, I reckon we're using jsify_scope to produce the code for the body but we should be able to add a boolean param to turn off generating the surrounding braces, or something to that effect

Reply:
I've actually changed `jsii_scope` to `jsii_scope_body` so it's just the body now.